utilities/oadp.py

```
@@ -137,3 +179,172 @@ def create_rhel_vm(
+        **kwargs (Any):
+            Additional keyword arguments passed to the base ``Restore`` class.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        client: DynamicClient,
+        namespace: str = ADP_NAMESPACE,
+        included_namespaces: list[str] | None = None,
+        backup_name: str | None = None,
+        teardown: bool = True,
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)